money/money.py

```diff
@@ -168,11 +191,38 @@ def _assert_same_currency(self, other: 'Money') -> None:
 
     @staticmethod
     def _round(amount: Decimal, currency: Currency) -> Decimal:
+        rounding_per_operation = Money.rounding_per_operation
+
+        if not rounding_per_operation:
+            return amount
+
+        rounding_type = Money.rounding_type
+
         sub_units = CurrencyHelper.sub_unit_for_currency(currency)
         # rstrip is necessary because quantize treats 1. differently from 1.0
         rounded_to_subunits = amount.quantize(Decimal(str(1 / sub_units).rstrip('0')),\
-                                              rounding=ROUND_HALF_UP)
+                                              rounding=rounding_type)
         decimal_precision = CurrencyHelper.decimal_precision_for_currency(currency)
         return rounded_to_subunits.quantize(\
                    Decimal(str(1 / (10 ** decimal_precision)).rstrip('0')),\
-                   rounding=ROUND_HALF_UP)
+                   rounding=rounding_type)
+
+    @classmethod
+    def set_rounding_per_operation(cls, value: bool) -> None:
+        """Set the whether to apply rounding after certain operations class-wide"""
+
+        if value is not True and value is not False:
+            raise TypeError
+
+        Money.rounding_per_operation = value
+
+    @classmethod
+    def set_rounding_type(cls, value: DecimalRoundingType) -> None:
+        """If ``rounding_per_operation`` is set, this determines the rounding method to use"""
+
+        if value not in [ROUND_CEILING, ROUND_DOWN, ROUND_FLOOR,
```

Comment:
can you avoid having to re-list all of these here? it seems like it could get easy for them to get out of sync with the values of DecimalRoundingType